Dockerfile

```diff
@@ -10,26 +10,38 @@ RUN sh ./ffmpeg-fetch.sh
 # Create our Ubuntu 22.04 with node 16.14.2 (that specific version is required as per: https://stackoverflow.com/a/72855258/8088021)
 # Go to 20.04
 FROM ubuntu:20.04 AS base
+ARG TARGETPLATFORM
 ARG DEBIAN_FRONTEND=noninteractive
 ENV UID=1000
 ENV GID=1000
 ENV USER=youtube
 ENV NO_UPDATE_NOTIFIER=true
 ENV PM2_HOME=/app/pm2
 ENV ALLOW_CONFIG_MUTATIONS=true
+# Directy fetch specific version
+## https://deb.nodesource.com/node_16.x/pool/main/n/nodejs/nodejs_16.14.2-deb-1nodesource1_amd64.deb
 RUN groupadd -g $GID $USER && useradd --system -m -g $USER --uid $UID $USER && \
     apt update && \
     apt install -y --no-install-recommends curl ca-certificates tzdata && \
-    curl -fsSL https://deb.nodesource.com/setup_16.x | bash - && \
-    apt install -y --no-install-recommends nodejs && \
-    npm -g install npm n && \
-    n 16.14.2 && \
     apt clean && \
     rm -rf /var/lib/apt/lists/*
+RUN case ${TARGETPLATFORM} in \
+         "linux/amd64")  NODE_ARCH=amd64   ;; \
+         "linux/arm")    NODE_ARCH=armhf  ;; \
+         "linux/arm/v7")    NODE_ARCH=armhf  ;; \
+         "linux/arm64")  NODE_ARCH=arm64  ;; \
+    esac \ 
+    && curl -L https://deb.nodesource.com/node_16.x/pool/main/n/nodejs/nodejs_16.14.2-deb-1nodesource1_$NODE_ARCH.deb -o ./nodejs.deb && \
```

Comment:
it doesn't install nodejs twice, first from repo that update version, then install v16. Or go to repo version? 🤔 Not really sure if it will trigger err 243 again